src/main/java/hudson/plugins/view/dashboard/stats/StatSlaves.java

```diff
@@ -51,9 +50,9 @@ private void countAgents(Computer[] computers) {
     }
 
     private int countRunningJobs(Jenkins j) {
-      List<Job> jobs = j.getAllItems(Job.class);
+      // TODO: Might be a faster way to get this info from the computers
       int countRunningJobs = 0;
-      for (Job job : jobs) {
+      for (Job job : j.allItems(Job.class)) {
```

Comment:
Possible do `ACL.as(ACL.SYSTEM)` here since we only return a count so permissions aren't all that important